demo/run-docker.sh

```diff
@@ -10,10 +10,19 @@ COMPOSE_YAML="$BASEDIR/demo/docker-compose.yaml"
 
 # Sanity check to prevent accidentally tripping oneself with an existing demo
 if ( docker-compose -f $COMPOSE_YAML ps | grep hydra-node > /dev/null 2>&1 ); then
-  echo "Demo already in progress, exiting"
+  echo >&2 -e "# Demo already in progress, exiting"
+  echo >&2 -e "# To stop the demo use: docker-compose down"
```

Comment:
question: If we are explicit on the `$COMPOSE_YAML` because we assume the script can be run from elsewhere, then we should also add the `-f ...` argument to the message.

Reply:
Good point. I would tend to `cd ${BASEDIR}/demo` though to not complicate the shutdown command.

Reply:
Did this now in f506e208c7cd5e0bebda0469743ff74c20f0abd2